File: VB/Models/Order.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace T223550.Models
	Public Class Order
		Public Property ID() As Integer
		Public Property Customer() As Customer
		Public Property OrderDate() As Date
		Public Overrides Function ToString() As String
			Return String.Format("{0} - {1:d}", ID, OrderDate)
		End Function
	End Class
End Namespace

File: CS/T223550/Models/Order.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace T223550.Models {
    public class Order {
        public int ID { get; set; }
        public Customer Customer { get; set; }
        public DateTime OrderDate { get; set; }
        public override string ToString() {
            return string.Format("{0} - {1:d}", ID, OrderDate);
        }
    }
}
